nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -7,7 +7,7 @@ In addition to numerous new and upgraded packages, this release has the followin
 ## Highlights {#sec-release-21.11-highlights}
 
 - PHP now defaults to PHP 8.0, updated from 7.4.
-- kOps now defaults to 1.21.0, which uses containerd as the default runtime.
+- kOps now defaults to 1.21.1, which uses containerd as the default runtime.
```

Comment:
Nit: Would be nice to add an empty line like any other items.

```suggestion

- kOps now defaults to 1.21.1, which uses containerd as the default runtime.
```

Reply:
When I originally added this there weren't any items after. Glancing at the rest of the document, it seems an extra blank line is the standard. Thanks for pointing this out.